### ISSUE TITLE
[Bug] incorrect break sound when voxel only gets deleted

### ISSUE THREAD

**Issue by Please-Pick-a-Name** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#182]
### Describe the bug

It seems that when a makehole only deletes voxel, it will play some sort of default break sound
this works normaly when you actually break off voxel from its parent body

i recorded some vid to help me explain issue

https://discord.com/channels/760105076755922996/768940642767208468/957039417443377194
https://discord.com/channels/760105076755922996/768940642767208468/957047068764024913
https://discord.com/channels/760105076755922996/768940642767208468/957054968253268058

### Steps to reproduce the behavior

1. break vox without anything voxel breakoff
2. hear incorrect sound

### Expected behavior

proper breaking sound corresponding to their material

### Environment

- Teardown Version: 0.9.5
- OS: Windows
- GPU:
- Graphics driver version:
- CPU:

### Additional context

i think i have heard this break sound for a very long time now, it is just now that i realize that its probably a bug, wow

i wonder how many more issues like this is there in the game that we thought is completely normal

**Comment by tuxedolabs** [comment]
Correct! I determine break sound based on the remaining pieces. If there is none I think wood is used as default. I should probably change this at some point, but it's not a priority. Thanks for reporting!

**Comment by Please-Pick-a-Name** [comment]
> Correct! I determine break sound based on the remaining pieces. If there is none I think wood is used as default. I should probably change this at some point, but it's not a priority. Thanks for reporting!
:pleading_face:

**Comment by YuLun-bili** [comment]
added the `dennis/wishlist` label

**Comment by gaberundlett-tuxedo** [comment]
I tried finding this in code but I couldn't figure out where the "fallback to wood" is. I'll have to chat with Dennis.

**Comment by Please-Pick-a-Name** [comment]
> I tried finding this in code but I couldn't figure out where the "fallback to wood" is. I'll have to chat with Dennis.

i’m not sure what the specific break sound is, but it is the classic crunch sound that you must have heard before. maybe it’s dirt? idk, but it certainly don’t care about what voxel material was actually destroyed
